pom.xml

```diff
@@ -34,7 +33,8 @@
 
 	<scm>
 		<connection>scm:git:git@github.com:javamelody/alfresco-javamelody.git</connection>
-		<developerConnection>scm:git:git@github.com:javamelody/alfresco-javamelody.git</developerConnection>
+		<developerConnection>
```

Comment:
why reformating ?

Reply:
my default IDE formatting rule for XML files...